### DISCUSSION TITLE
Managing User-Team Relations and Invites

### DISCUSSION THREAD

**Discussion by JessedeJonge** [pocketbase/pocketbase#2872] [Q&A]
I have a database with four tables: users, teams, games, and teamInvites. The relationships between these tables are as follows: a user can belong to multiple teams, a team can have multiple users, and a team can have multiple games. The teamInvites table is a view that selects metadata about the teams and is restricted via an inviteCode check, which is a property of teams.

Currently, I am working on implementing a functionality where a user can join a team using an inviteCode. I want to ensure that the user-team relations are updated correctly and efficiently.

My question is: What is the best approach to create and update the relations between users and teams using the JavaScript SDK? Should I retrieve the existing values of the teams.members (users) relation, append the user ID of the user attempting to join the team, or is there a safer and better solution for this? Perhaps using a joined table would be a better option?

I would appreciate any guidance or tips on how to handle this situation effectively.

Thank you.

**Comment by j6k4m8** [comment]
#4475 might be useful for developers encountering a similar issue!
